message MsgClient {
	required string message = 1; 
	required string other_uid = 2;
	required string select_type = 3;
};